### PR TITLE
prompted confession

### PR DESCRIPTION
**Pull request by astrofinix**
added a prompted confession via "/prompt" and reply function to it via "/reply"

added an increment counter for the prompted confession questions.

### CODE REVIEW

File: bot/database.py [hunk 1]
```diff
@@ -100,4 +100,10 @@ async def increment_counter(self) -> int:
         worksheet = await self._get_worksheet(1)
         cell = await worksheet.cell(1, 2)
         await worksheet.update_cell(1, 2, int(cell.value) + 1)
+        return int(cell.value)
+
+    async def question_increment_counter(self) -> int:
```
Comment (brymer-meneses) [inline bot/database.py]: Rename this to `increment_prompted_question_counter `

**Review by astrofinix — COMMENTED** [review]
updated the function names

**Comment by brymer-meneses** [comment]
@fphinix Paayos para ma-merge ko

**Review by danieldavidbador — APPROVED** [review]

**Comment by brymer-meneses** [comment]
I think reply_with_nickname can be merged to the reply command, by making the nickname parameter optional.

**Review by astrofinix — COMMENTED** [review]
noice

**Pull request merged**